openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip_jsoniter_scala3/src/main/scala/injections/Types.scala

```diff
@@ -0,0 +1,6 @@
+package injections
+
+object Types {
+  type A[T] = cats.effect.IO[T]
```

Comment:
Just a dummy type, doesn't really matter for the purposes of generated code validation, but would probably be a Kleisli or something IRL